.rubocop-http---shopify-github-io-ruby-style-guide-rubocop-yml

```diff
@@ -20,7 +20,7 @@ Style/Alias:
   - prefer_alias
   - prefer_alias_method
 
-Layout/AlignHash:
+Layout/HashAlignment:
```

Comment:
When I run  `bundle exec rubocop` locally I get 
`Warning: unrecognized cop Sorbet/ConstantsFromStrings found in .rubocop.yml` plus 23 failures all but 1 in serial_deploy_test.rb. CI has a bunch of warnings but no failures. Any idea what's up?

Reply:
I think it's a mismatch between the rubocop config and the rubocop version. I'm not sure how to prevent that. Right now we pin the rubocop gem version to make sure our CI doesn't leap ahead of the style guide, but the reverse can happen: the styleguide gets updates in a way that's not compatible with our (now-old) pinned version. I was hoping we might be able to change force our CI invocation to use the committed copy of the styleguide, but it looks like that isn't possible. Rubocop will try to refresh the cache if it is > 24 hours old.

In any case, I'll solve the immediate problem by bumping rubocop, updating the committed file again, and removing some rules we didn't need anymore while I'm at it.